utils/build-script

```diff
@@ -955,7 +955,7 @@ details of the setups of other systems or automated environments.""")
         help="Pass through extra options to CMake in the form of comma "
              "separated options '-DCMAKE_VAR1=YES,-DCMAKE_VAR2=/tmp'. Can be "
```

Comment:
Is the comment still accurate?  It seems like the list is space-separated, or rather, shell-script-style.